### ISSUE TITLE
Support -> Send Email Doesn’t Work 

### ISSUE THREAD

**Issue by jubishop** [gdavis/Mimir-Feedback#258]
### Are you sure this is a new issue?

- [X] I have confirmed this is a new issue

### iOS Version

17.4.1

### Device Model

iPhone 15 Pro

### App Version

1.4.3

### What happened?

When I go to settings - support - send email it says it can’t.  I copy the message to the clipboard which is:

Cannot Send Email
This device cannot send email. Please check the Settings app and make sure your email account is configured correctly.

Device Info:
App Version:    1.4.3
Build:          137
Device:         iPhone
Device Model:   iPhone
OS Version:     17.4.1
Locale:         en_US
Language:       en

I do have a default mail client set in settings (it’s Hey), so this should work iiuc.  

### Bug prevalence

Always 

### Add screenshots or video.

![IMG_1552](https://github.com/gdavis/Mimir-Feedback/assets/13165/7c81c13c-ab35-4e1c-a127-c17ab977f960)

**Comment by gdavis** [comment]
Hmm this is an interesting issue. This alert appears after I perform a check with iOS to see if the device can send email. Are you able to click on links like mailto:support@mimirpodcasts.app and get the mail composer to appear?

**Comment by jubishop** [comment]
Yeah clicking that link brings up the Hey email composer on my iPhone just fine .

**Comment by g3rv4** [comment]
I could repro this. fwiw, I've deleted the Mail application and use GMail.

When I click "copy to clipboard", I don't get the email on the clipboard, but this:

```
Cannot Send Email
This device cannot send email. Please check the Settings app and make sure your email account is configured correctly.

Device Info:
App Version:    1.4.3
Build:          137
Device:         iPhone
Device Model:   iPhone
OS Version:     17.6.1
Locale:         en_US
Language:       en
```
